modules/dsme_dbus.c

```diff
@@ -1210,6 +1211,9 @@ manager_handle_introspect(DsmeDbusManager *self, DBusMessage *req)
         fclose(file);
     free(data);
 
+    if( children )
+        keys_free(children);
```

Comment:
It is okay to call keys_free(NULL). So the conditional here should be dropped.

[that is: in comparison to surrounding code this is like free(), not like fclose()]

Reply:
Sure.